src/test/java/seedu/address/model/appointment/AppointmentTest.java

```diff
@@ -38,6 +39,36 @@ void setUp() {
             person1, new Sickness("Cold"), new Medicine("Aspirin"));
         appointment3 = new Appointment(new AppointmentType("Followup"), dateTime2,
             person2, new Sickness("Flu"), new Medicine("Tamiflu"));
+
```

Comment:
Should there be a closing `}` for `setUp()`? The syntax errors are causing the build checks to fail.

Reply:
Nice catch, thanks